com.unity.robotics.urdf-importer/Tests/Runtime/Extensions/UrdfCollisionsExtensionsTests.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEditor;
+using UnityEngine.TestTools;
+using RosSharp.Urdf;
+using Object = UnityEngine.Object;
+using Collision = RosSharp.Urdf.Link.Collision;
+using Geometry = RosSharp.Urdf.Link.Geometry;
+using Box = RosSharp.Urdf.Link.Geometry.Box;
+
+namespace RosSharp.Urdf.Tests
+{
+    public class UrdfCollisionsExtensionsTests
+    {
+        Geometry geometryBox = new Geometry(box: new Box(new double[] {1, 1, 1}));
```

Comment:
nit: Should the geometryBox be shared between tests, or created before each test run?

Reply:
would it be better to set it up in `[Setup]` or as a `TestCaseData` instead of as a member to this test class?

Reply:
Yes. It is a good practice to set up testing data per test, instead of sharing them.